lib/rules/heuristic/group.js

```diff
@@ -1,45 +0,0 @@
-
```

Comment:
Chatted with @deniak: the new method `extractMetadata()` will be used instead of these old rules that return just `info()` messages.